_data/karthik1125/karthik1125.md

```diff
@@ -0,0 +1,7 @@
+---
+name: KARTHIKEYA MANTHRI
+institution: INDIAN INSTITUTE OF INFORMATION TECHNOLOGY SRI CITY
+profile_pic: Karthik_octocat.png
+quote: I have learned from the bad times and was humbled by the good. Thank you for all of the great life lessons.
```

Comment:
Quote must **not** be longer than 100 characters, yours is **107** characters.

Reply:
Sorry for that, I've updated the same. Thanks for pointing it out!